the-other-feature-description.md

```diff
@@ -0,0 +1,17 @@
+Beef soft tacos
+Ingredients
+seasoned beef
+Soft corn tortillas
+salsa
+guacamole
+Shredded Mexican cheese
+Shredded lettuce
+Sour cream
+Method
```

Comment:
There could be empty line before 'Method'

Reply:
True!